README.md

```diff
@@ -366,29 +352,13 @@ ____   ____.__     .___                     __          ____   ____
 ╔═════════════════════════╤════════════════════════════════════════════════════════════════════════════════╗
 ║ Name                    │ Description                                                                    ║
 ╟─────────────────────────┼────────────────────────────────────────────────────────────────────────────────╢
-║ start                   │ Starts dev server and watch task                                               ║
-╟─────────────────────────┼────────────────────────────────────────────────────────────────────────────────╢
-║ watch║                  │ watches for changes on the plugin files and executes the appropriate tasks     ║
-╟─────────────────────────┼────────────────────────────────────────────────────────────────────────────────╢
-║ build                   │ This task builds the plugin                                                    ║
+║ build                   │ This task runs karma tests and builds the plugin                               ║

```

Comment:
Let's list the operations in the order they happen: "This task builds the plugin and runs karma tests.

Reply:
Done